steps/variables-libvirt.tf

```diff
@@ -1,6 +1,6 @@
-variable "tectonic_libvirt_ssh_key" {
+variable "tectonic_libvirt_uri" {
   type        = "string"
-  description = "Contents of an SSH key to install for the core user"
+  description = "libvirt connection URI"
```

Comment:
Add `default = "qemu:///system"`?

Reply:
This is a required field.

Reply:
> This is a required field.

But previously we'd hard-coded `qemu://system` in a number of places.  Do we need to require it if we have a value that usually works that we can use as a default?  Or is that default value not as reliable as it seemed?

Reply:
I think we are going to see too many developers get hit by this if we have a silent default. OSX users, for example, will probably end up using `vmware://system` (or whatever the URI is). I'd rather be explicit about this for now.